easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.10.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,39 @@
+name = 'TensorFlow'
+version = '1.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_remove-msse-hardcoding.patch',
+    '%(name)s-%(version)s_dont_expand_cuda_cudnn_path.patch',
```

Comment:
@akesandgren Please add back `TensorFlow-1.5.0_swig-env.patch`, it seems to be required to fix a problem I'm running into without this, i.e. the build failing with `swig failed: error executing command`